scripts/setup-local-environment.sh

```diff
@@ -69,5 +69,5 @@ Done building the local images.
 
 To run the tests enter:
 
-scripts/testing/test-software.sh
+scripts/testing/test-docker-software.sh
```

Comment:
I think this is correct with the changes you've made:

```suggestion
scripts/testing/run_local_tests.sh
```